content/docs/reactivesearch/v3/search/searchbox.md

```diff
@@ -204,10 +226,35 @@ Example uses:
             minHits: 5,
             minChars: 3,
             index: "good-books-ds",  // further restrict the index to search on
+            sectionLabel: '<h3>Index suggestions</h3>' 
+        }}
+    />
+```
+
```

Comment:
@mohdashraf010897 Include the following note for featured suggestions:

Featured suggestions allow creating autocomplete experiences with user-defined suggestions. They're specified using the [Featured Suggestions API](https://api.reactivesearch.io/#337cdab6-d06c-4319-8c51-51e9ff0c1266), introduced in 8.1.0. This is a beta API and subject to change.

Reply:
https://github.com/appbaseio/Docs/pull/312/commits/f66ab86c0858b709fed2cf152dbc3f0a22796b36